modules/ROOT/pages/install.adoc

```diff
@@ -0,0 +1,229 @@
+
+== Supported platforms
+
+The {product} supports the following operating systems:
+
+[cols="1,2"]
+|===
+|Operating System |Architecture
+
+|Linux
+|`x86_64`, `arm64`
+
+|macOS
+|`x86_64`, `arm64`
+
+|Microsoft Windows
+|`x86_64`
+|===
+
+== Install the CLI
+
+Select one of the following installation methods and follow the steps to install the {product}.
+
+[tabs,sync-group-id=platforms]
+======
+macOS and Linux::
++
+Select one of the following installation methods to install the {product} on macOS and Linux.
+If you prefer to use a package manager, follow the Homebrew instructions instead.
++
+[tabs]
+=====
+Scripted installation (recommended)::
++
+--
+. Run the following command to install the latest version of the {product} to the default location:
++
+[source,shell]
+----
+curl -sSL ibm.biz/astra-cli | sh
+----
++
+.Install to a custom location
+[%collapsible]
+====
+To install the {product} to a custom location, export the `ASTRA_HOME` environment variable before running the installation command:
+
+[source,bash,subs="+quotes"]
+----
+export ASTRA_HOME=**INSTALLATION_DIRECTORY**
+----
+
+Replace `**INSTALLATION_DIRECTORY**` with the full path to your preferred installation directory.
+====
+
+. Add the following lines to your shell profile (for example, `~/.zprofile` or `~/.bash_profile`) to make the `astra` command executable from any directory:
++
+[source,bash]
+----
+# Astra CLI shell configuration
+eval "$(~/.astra/cli/astra shellenv)"
+----
+
+. Reload your shell profile or open a new terminal window.
+
+. Run the `astra` command from any directory to verify successful installation:
++
+[source,shell]
+----
+astra
+----
++
+The response includes available commands and options for the {product}.
+--
+
+Binary installation::
++
+--
+. Download the appropriate binary archive for your OS and CPU architecture from https://github.com/datastax/astra-cli/releases[GitHub].
+
+. Extract the archive into the directory where you want to install the {product}.
+For example:
++
+[source,shell,subs="+quotes"]
+----
+sudo tar -xzvf astra-**OS**-**ARCHITECTURE**.tar.gz -C **INSTALLATION_DIRECTORY**
+----
++
+Replace the following:
++
+* `**OS**`: Your operating system (`macos` or `linux`).
+* `**ARCHITECTURE**`: Your CPU architecture (`x86_64` or `arm64`).
+* `**INSTALLATION_DIRECTORY**`: The full path to your preferred installation directory.
+
+. Add the following lines to your shell profile (for example, `~/.zprofile` or `~/.bash_profile`) to make the `astra` command executable from any directory:
++
+[source,bash,subs="+quotes"]
+----
+# Astra CLI shell configuration
+eval "$(**PATH_TO_BINARY** shellenv)"
+----
++
+Replace `**PATH_TO_BINARY**` with the full path to the `astra` executable in the installation directory.
+
+. On macOS: Delete the `com.apple.quarantine` attribute from the `astra` binary to remove https://support.apple.com/guide/security/gatekeeper-and-runtime-protection-sec5599b66df/web[macOS Gatekeeper restrictions]:
```

Comment:
Is this required? will the Astra CLI not work if you don't remove the restrictions? Or do you just get the security warning?

```suggestion
. Required for macOS: Delete the `com.apple.quarantine` attribute from the `astra` binary to remove https://support.apple.com/guide/security/gatekeeper-and-runtime-protection-sec5599b66df/web[macOS Gatekeeper restrictions]:
```

Reply:
The alert blocks `astra` from running entirely until you remove the restriction, either with the supplied command or via System Settings.